packages/remix-react/components.tsx

```diff
@@ -945,7 +945,6 @@ let FormImpl = React.forwardRef<HTMLFormElement, FormImplProps>(
                 event.preventDefault();
 
                 submit(clickedButtonRef.current || event.currentTarget, {
-                  method,
```

Comment:
Shouldn't this be `method: formMethod` instead, which is sanitized above?

Reply:
I don't think so, the `method` props of the `FormImpl` component is the `method` props of the `Form` element. 
Sanitization won't solve the issue here, in both cases it will pass `'get'` to the `submit` function `options`, which will then be used as the default value for the form submission despite the `formmethod` attribute. 

If I understand it well, the `method` in the submit function `options` is not intended to be inferred from the DOM elements and their attributes (this is handled in the definition of the `useSubmitImpl` hook), but rather to overwrite the method in case of imperative submissions (e.g. using the `useSubmit` hook).